Makefile

```diff
@@ -29,16 +29,19 @@ dev: checklist check explaintest gogenerate br_unit_test test_part_parser_dev ut
 	@>&2 echo "Great, all tests passed."
 
 # Install the check tools.
-check-setup:tools/bin/revive
+check-setup:lint
 
 check: check-file-perm check-parallel lint tidy testSuite errdoc bazel_golangcilinter bazel_all_build
 
 fmt:
 	@echo "gofmt (simplify)"
 	@gofmt -s -l -w $(FILES) 2>&1 | $(FAIL_ON_STDOUT)
 
-check-static: tools/bin/golangci-lint
-	GO111MODULE=on CGO_ENABLED=0 tools/bin/golangci-lint run -v $$($(PACKAGE_DIRECTORIES)) --config .golangci.yml
+check-static:
```

Comment:
Is anyone actually using `check-static`? I don't see any reference in the script.

Reply:
we will remove it If none really use it.